core-web/libs/ui/src/lib/components/dot-workflow-actions/dot-workflow-actions.component.ts

```diff
@@ -34,71 +39,121 @@ const InplaceButtonSizePrimeNg: Record<ButtonSize, string> = {
     changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class DotWorkflowActionsComponent implements OnChanges {
+    /**
+     * List of actions to display
+     *
+     * @type {DotCMSWorkflowAction[]}
+     * @memberof DotWorkflowActionsComponent
+     */
     @Input({ required: true }) actions: DotCMSWorkflowAction[];
```

Comment:
I suggest using Signal Input here and in other inputs

Reply:
Done!